compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/FlushCoroutineDispatcher.skiko.kt

```diff
@@ -87,4 +92,12 @@ internal class FlushCoroutineDispatcher(
             isPerformingRun = false
         }
     }
+
+    @OptIn(ExperimentalCoroutinesApi::class)
+    override fun scheduleResumeAfterDelay(timeMillis: Long, continuation: CancellableContinuation<Unit>) {
+        scope.launch {
+            kotlinx.coroutines.delay(timeMillis)
```

Comment:
we launch in a scope to deleate the delay to an actual Delay prvided by scope.coroutineContext (in tests it will be UnconfinedTestDispatcher)